mobile/src/bundles/talent/store/actions.ts

```diff
@@ -58,13 +57,16 @@ const updateOnboardingData = createAsyncThunk<
 });
```

Comment:
`return stepPayload as unknown as UserDetailsGeneralResponseDto;`
please, delete this 'unknown' from previous PR.

Reply:
I tried, but if I delete it get an error

Reply:
which error

Reply:
Everything is good now